app/javascript/controllers/clipboard_controller.js

```diff
@@ -3,7 +3,18 @@ import { Controller } from "@hotwired/stimulus"
 export default class extends Controller {
   static values = { copy: String, tooltip: String }
 
+  originalStyleDisplay
```

Comment:
What is this?  I guess I don't usually see stimulus controllers defining instance variables outside of the `connect()`.

Reply:
Needed a class property that could be referenced and updated in multiple class methods overtime.